shared/utils/count_oval_objects.py

```diff
@@ -111,7 +111,19 @@ def main():
         rule_id = rule.attrib['id']
         oval_refs = []
         for ref in rule.findall(".//check-content-ref"):
-            oval_name = ref.attrib['name']
+
+            # Skip remotely referenced OVAL checks since they won't have the
+            # 'name' attribute set (just 'href' would be set in that case)
+            try:
+                oval_name = ref.attrib['name']
+            except KeyError:
+                if ref.attrib['href'] is not None:
```

Comment:
Shouldn't this be `"href" not in ref.attrib`? `ref.attrib['href']` will throw KeyError again I think if "href" isn't among the keys.